boxes.cc

```diff
@@ -31,14 +34,23 @@ using namespace benchmark;
 // Boxes:
 // Spawn a single box and record accuracy for momentum and enery
 // conservation
-void BoxesTest::Boxes(const std::string &_physicsEngine
-                    , double _dt
-                    , int _modelCount
-                    , bool _collision
-                    , bool _complex)
+void BoxesTest::Boxes(const std::string& _physicsEngine, double _dt, int _modelCount, bool _collision, bool _complex)
 {
+  ASSERT_GT(_modelCount, 0);
+
+  std::stringstream command;
+  command << "erb "
+          << "engine=" << _physicsEngine << " model_count=" << _modelCount << " collision=" << _collision
+          << " complex=" << _complex << " dt=" << _dt << " " << WORLDS_FILE_PATH << "/boxes.world.erb > "
+          << WORLDS_FILE_PATH << "/boxes.world";
+
+  // creating model with desired configuration
+  auto model_check = system(command.str().c_str());
+  // checking if model is created
+  ASSERT_EQ(model_check, 0);
+
   // Load a blank world (no ground plane)
-  Load("worlds/blank.world", true, _physicsEngine);
+  Load("worlds/boxes.world", true, _physicsEngine);
```

Comment:
```suggestion
  Load(WORLDS_FILE_PATH "/boxes.world", true, _physicsEngine);
```
you can tell the loader where the world file is so you don't have to set the `GAZEBO_RESOURCE_PATH`